lib/smart_answer/calculators/marriage_abroad_calculator.rb

```diff
@@ -223,5 +223,13 @@ def requires_7_day_notice?
     def same_sex_alt_fees_table_country?
       @data_query.ss_alt_fees_table_country?(ceremony_country, self)
     end
+
+    def outcome_path_when_resident_in_uk
+      [
+        '', 'marriage-abroad', 'y',
+        @ceremony_country, 'uk',
+        @partner_nationality, @sex_of_your_partner
+      ].join('/')
+    end
```

Comment:
Can you explain why you chose not to use `State#current_path` here? I think it probably does make sense, but I think at the very least you should explain the reasoning in the commit note.